rasa/train.py

```diff
@@ -551,6 +551,9 @@ async def _train_core_with_validated_data(
         )
 
         if model_to_finetune:
+            rasa.shared.utils.common.mark_as_experimental_feature(
```

Comment:
seems this module isn't imported yet.

Reply:
fixed

Reply:
Hmm, but I tested it and it worked 🤔

Reply:
It's my IDE which is outraged
![Screenshot 2020-12-14 at 17 06 48](https://user-images.githubusercontent.com/7667273/102104690-c76a9200-3e2e-11eb-960c-a6a7eef44997.png)